exercises/phone-number/src/example.c

```diff
@@ -49,23 +51,10 @@ char *phone_number_clean(const char *input)
          strcpy(output, INVALID_NUMBER_RESULT);
       }
    }
-   return output;
-}
 
-char *phone_number_get_area_code(const char *input)
-{
-   char *output = phone_number_clean(input);
-   output[3] = '\0';
-   return output;
-}
-
-char *phone_number_format(const char *input)
-{
-   char *cleaned_input = phone_number_clean(input);
-   char *output = calloc(FORMATTED_LENGTH + 1, sizeof(char));
+   if (strcmp(output, INVALID_NUMBER_RESULT) != 0)
```

Comment:
Just a micro optimization so that I don't check for the if the number is less than '2' when I already know it is. It may actually be less efficient looking at it now, since its calling strcmp and iterating over the string.

Reply:
Right, that's what I was thinking. Let's go ahead and omit it and I'll give this my thumbs up :+1:

Reply:
Done...